benefits/locale/es/LC_MESSAGES/django.po

```diff
@@ -28,6 +28,9 @@ msgstr ""
 "may need to renew. Using this benefit means your new transit fare is half of "
 "the standard fare."
 
+msgid "eligibility.pages.start.mst_login.title"
+msgstr "TODO: Older Adult benefit information"
+
 msgid "eligibility.pages.start.mst_login.headline"
 msgstr ""
 "Conecta tu tarjeta bancaria para tu descuento en el transporte público con "
```

Comment:
This should be `"TODO: You selected an Older Adult benefit."` -- this is the old text, may be confusing / seem like it is already translated.

Reply:
Resolved: https://github.com/cal-itp/benefits/pull/1093/commits/2d40b0a57f0017dd3c516c867628ba0d902fcf58